pyproject.toml

```
@@ -18,7 +18,7 @@ authors = [
 ]
 license = { text = "BSD 2-clause" }
 requires-python = ">=3.9"
-dependencies = ["numba>=0.59.1"]
+dependencies = ["numba>=0.59.1,<0.61.0"]
```

Please disallow alphas in upper bounds.
```suggestion
dependencies = ["numba>=0.59.1,<0.61.0a0"]
```